deno/corejs/index.js

```diff
@@ -969,7 +969,7 @@ var createElement = __webpack_require__(39);
 
 // Thank's IE8 for his funny defineProperty
 module.exports = !DESCRIPTORS && !fails(function () {
-  // eslint-disable-next-line es/no-object-defineproperty -- requied for testing
+  // eslint-disable-next-line es/no-object-defineproperty -- required for testing
```

Comment:
Some projects don't want PRs to include changes to generated files. I'm not sure if this is one of them.

Let me know.

Reply:
I don't care - this file will be automatically regenerated on the next release.